### PR TITLE
NOISSUE- Clients Integration

### PR DESCRIPTION
**Pull request by rodneyosodo**
Signed-off-by: rodneyosodo <blackd0t@protonmail.com>

### What does this do?
Add documentation for clients integration

### Which issue(s) does this PR fix/relate to?
NOISSUE

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
Yes

### Did you document any new/modified functionality?
Yes

### Notes
To be merged after https://github.com/mainflux/mainflux/pull/1759

### CODE REVIEW

File: docs/authentication.md [hunk 1]
```diff
@@ -1,29 +1,26 @@
 # Authentication
 
 ## User authentication
+
 For user authentication Mainflux uses Authentication keys.
-There are three types of authentication keys:
+There is one two types of authentication keys:
```
Comment (drasko) [inline docs/authentication.md]: one or two?

**Review by drasko — CHANGES_REQUESTED** [review]

File: docs/authentication.md (hunk 1 above)
Comment (rodneyosodo) [reply docs/authentication.md]: two

**Comment by drasko** [comment]
@rodneyosodo please resolve conflicts

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**